src/service/action-impl.js

```diff
@@ -163,9 +167,6 @@ export class ActionService {
    */
   installActionHandler(target, handler) {
     const debugid = target.tagName + '#' + target.id;
-    user().assert(target.id && target.id.substring(0, 4) == 'amp-',
```

Comment:
(1) This should be a `dev().assert()`
(2) Why not also check here `supportedActions`? This way you get an early notification as a developer that an element (at least) must be whitelisted.